### PR TITLE
add missing articles to Create Private Key section for clarity

### PR DESCRIPTION
**Pull request by jonassteinberg1**
Adding missing articles like 'a' in a couple places to make section more readable.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info https://github.com/kubernetes/website/pull/29728#issuecomment-920900768

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -200,12 +200,12 @@ To allow signing a CertificateSigningRequest:
 
 A few steps are required in order to get a normal user to be able to
 authenticate and invoke an API. First, this user must have certificate issued
-by the Kubernetes cluster, and then present that certificate to the Kubernetes API.
+by the Kubernetes cluster and then present that certificate to the Kubernetes API.
```
Comment (RinkiyaKeDad) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:203]: Should we maybe reword this entire section to something like:

```
To get a normal user to be able to authenticate and invoke an API, the user must have a certificate issued by the Kubernetes cluster, and then they must present that certificate to the Kubernetes API.
```

**Review by RinkiyaKeDad — COMMENTED** [review]
Thanks for the PR, left a couple of suggestions :)

**Comment by sftim** [comment]
Hi @jonassteinberg1 

Would you be willing to rebase this to incorporate changes since you opened this PR?

**Comment by jimangel** [comment]
@jonassteinberg1 it appears this PR needs merge conflict resolution for `content/en/docs/reference/access-authn-authz/certificate-signing-requests.md` as well as a couple nits from @RinkiyaKeDad. Given the age of the PR without action, I'm going to close it, but feel free to reopen it when ready! If you have any questions, please reach out on [slack](https://kubernetes.slack.com/messages/sig-docs)! Thanks

/close
